tools/create-translation-issues.sh

```diff
@@ -0,0 +1,216 @@
+
+We would be happy if you updated the Jamulus software translations for **${LANG}** until **${DEADLINE}**.
+
+Please
+- Update your fork to current `master`,
+- Update translations using Qt Linguist in your fork,
+- Commit and push your changes and reference this Issue,
+- Open a Pull Request before ${DEADLINE}
+
+Suggested commit and PR message:
+```
+${TITLE}
+
+Fixes #<Insert this issue'"'"'s number here>
+```
+
+${EXTRA_TEXT}${MULTIPLE_TRANSLATORS_TEXT}
+
+Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/TRANSLATING.md).
+
+Thanks for contributing to Jamulus!'
+
+
+BODY_TEMPLATE_WEB='Hi ${SPLIT_TRANSLATORS},
+
+We are getting ready for the ${RELEASE} release and have created the [${TRANSLATE_BRANCH}](https://github.com/jamulussoftware/jamuluswebsite/tree/${TRANSLATE_BRANCH}) branch ([full diff](https://github.com/jamulussoftware/jamuluswebsite/compare/release..${TRANSLATE_BRANCH})).
+
+Because the English is still going through large changes, we are not expecting translations of anything other than the following pages:
+
+- [Home page](https://github.com/jamulussoftware/jamuluswebsite/blob/${TRANSLATE_BRANCH}/1-index.html)
+- [Setup](https://github.com/jamulussoftware/jamuluswebsite/blob/${TRANSLATE_BRANCH}/wiki/en/en-Setup.md) (renamed from Getting-Started)
+- Installation pages for each supported platform ([Windows](https://github.com/jamulussoftware/jamuluswebsite/blob/${TRANSLATE_BRANCH}/wiki/en/en-Installation-for-Windows.md), [Mac](https://github.com/jamulussoftware/jamuluswebsite/blob/${TRANSLATE_BRANCH}/wiki/en/en-Installation-for-Macintosh.md), [Linux](https://github.com/jamulussoftware/jamuluswebsite/blob/${TRANSLATE_BRANCH}/wiki/en/en-Installation-for-Linux.md))
+
+While you are of course welcome to translate more than this, please be aware that you may have to re-translate for the next release.
+
+We would be happy if you updated the translations for **${LANG}** until **${DEADLINE}**.
+
+Please
+
+- Start your work in your fork on a branch based on jamuluswebsite'"'"'s `${TRANSLATE_BRANCH}` branch.
+- Update the language-specific files using your favorite editor (or directly on Github),
+- Commit and push your changes to your fork,
+- Open a Pull Request with your translations to the **${TRANSLATE_BRANCH}** branch with the subject `${TITLE}`,
+- Link your PR(s) to this issue by including `Fixes #<Insert this issue'"'"'s number here>` in the PR content.
+
+${EXTRA_TEXT}${MULTIPLE_TRANSLATORS_TEXT}
+
+Suggested commit and PR message:
+```
+${TITLE}
+
+Fixes #<Insert this issue'"'"'s number here>
+```
+
+Feel free to use this Issue to discuss anything you need prior to making any PR (and in your native language if needed).
+
+Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamuluswebsite/blob/${TRANSLATE_BRANCH}/README.md#translating).
+
+Thanks for contributing to Jamulus!'
+
+
+if ! gh auth status &>/dev/null; then
```

Comment:
Purely as a coding style point: this active section of code is sandwiched between two passive declarations.  It could be overlooked.  I'd move it to the top (just after the `set -eu`).

Reply:
Yep, good point, have moved it to the top (but kept the argument check / Usage information before it).